rest/route/github.go

```diff
@@ -240,8 +240,22 @@ func (gh *githubHookApi) handleComment(ctx context.Context, event *github.IssueC
 		grip.Info(gh.getCommentLogWithMessage(event, "commit queue triggered"))
 
 		_, err := data.EnqueuePRToCommitQueue(ctx, evergreen.GetEnvironment(), gh.sc, createEnqueuePRInfo(event))
-		grip.Error(message.WrapError(err, gh.getCommentLogWithMessage(event, "can't enqueue on commit queue")))
-		return errors.Wrap(err, "enqueueing in commit queue")
+		if err != nil {
+			owner := event.Repo.Owner.GetLogin()
+			repo := event.Repo.GetName()
+			prNum := event.Issue.GetNumber()
+			grip.Error(message.WrapError(gh.displayCommitQueueEnqueueError(ctx, owner, repo, prNum, err), message.Fields{
```

Comment:
This is also going to post an error for checking if the project is tracking the commit queue to begin with https://github.com/evergreen-ci/evergreen/blob/91095164b387b5216b26082e0b830cdb38fc2b16/rest/data/commit_queue.go#L322-L321 which means we'll comment this on any 10gen project, which will be confusing, and it seems wrong to comment on disabled projects. Can we refactor to check this first or only comment if we return a bool?

Reply:
10gen projects aren't tracked in Evergreen; they use a CR bot to listen to the `evergreen merge` comment themselves, and then use the changes to add them to the commit queue for the mongodb/mongo project (which _is_ tracked) using the CLI. I wish very much that they had used a different comment but I think they wanted users to not have to consider what project they were in. This is what the #github-pr channel is for. Although this is a bit niche, this does effect a number of projects and would result in us posting a lot of unnecessary comments that are likely to cause confusion.

Reply:
Ah, I see... This is definitely not the reason I was expecting and I don't know if anyone knows this, so I'll try mentioning that the disabled commit queue with `evergreen merge` is a special case.